### DISCUSSION TITLE
World hopper whitelist/blacklist

### DISCUSSION THREAD

**Discussion by pettrs** [runelite/runelite#16084] [Ideas]
is it possible to add a blacklist of worlds to exclude from the world cycle when quick hopping with keybinds?

**Comment by samszotkowski** [comment]
I agree this would be useful.  The filters do a great job, but it would be a very nice QOL improvement to blacklist just a few worlds.  For example, Target Worlds have an additional dialog popup that causes you to interrupt your flow when hopping.  That issue could be solved by blacklisting worlds 318 and 319, for example.

Maybe a textbox for comma-separated world numbers could be a simple enough UI in the plugin settings.

**Reply by samszotkowski** [reply]
Turns out the [World Finder](https://github.com/Cgende/World-Finder) plugin is essentially the same as World Hopper, but contains a few more filters including Target Worlds.  Seems like these plugins are nearly identical and maybe could be consolidated one day.  See: https://github.com/Cgende/World-Finder/issues/1
